comps/dataprep/src/Dockerfile

```diff
@@ -1,6 +1,16 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+FROM python:3.11-slim AS models
+RUN useradd -m -s /bin/bash user && \
+    mkdir -p /home/user && \
+    chown -R user /home/user/
+RUN pip install "huggingface_hub[cli]"
+USER user
+WORKDIR /home/user
+ARG models="unstructuredio/yolo_x_layout unstructuredio/detectron2_faster_rcnn_R_50_FPN_3x unstructuredio/detectron2_mask_rcnn_X_101_32x8d_FPN_3x"
```

Comment:
We are not approved to distribute a model inside docker images, that will need special release process. So I don't suggest to do this.

Reply:
Are we allowed to include the nltk_data inside the container?

Reply:
First model seems to have Apache2 license in HF, but others are missing license info.

(Another issue with lawyers could be that legal status for models is still somewhat open, there are open legal cases debating copyright weight of the material models have been trained on etc.)

`nltk` project license is Apache2, but `nltk_data` project is missing license: https://github.com/nltk

If the data cannot be included into container itself, another possibility for offline use is having separate data downloader and separate volume for that data.

Reply:
Resolved. New patch exclude the model from the container.